colossus-metrics/src/main/scala/colossus/metrics/ConfigHelper.scala

```diff
@@ -38,6 +39,13 @@ object ConfigHelpers {
 
     def getScalaDuration(path: String): Duration = Duration(config.getString(path))
 
+    def getAddresses(path:String): Seq[String] = try {
+      config.getStringList(path).asScala
+    } catch {
+      case m: Missing => Seq("localhost")
+      case wt: WrongType => throw new WrongType(config.origin(), wt.getMessage)
```

Comment:
Seems like you are catching and then throwing the same exception. You don't need to catch it.